ci/test-conformance-eks.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
```

Comment:
Please use the `#!/usr/bin/env bash` shebang as it searches for bash in the PATH instead of using an absolute path

Reply:
will do

Reply:
Usually we'll add `set -exu`, too.